api/src/main/kotlin/com/google/devtools/ksp/processing/Resolver.kt

```diff
@@ -238,6 +238,15 @@ interface Resolver {
     @KspExperimental
     fun getDeclarationsFromPackage(packageName: String): Sequence<KSDeclaration>
 
+    /**
+     * Returns a sequence of subpackage names of the given [packageName].
```

Comment:
The implementation is only looking at immediate subpackage, rather than recursively searching deeper sub packages, it is worth pointing out in the API documentation here.

Reply:
oh I intended for it to recurse, would rather it not?

Reply:
Ah I guess I must have misunderstood the code, never mind.

Reply:
Maybe commenting that it is recursive then?